support/Prolog.YAML-tmLanguage

```diff
@@ -4,7 +4,7 @@ comment: This Source Code Form is subject to the terms of the Mozilla Public Lic
   one at http://mozilla.org/MPL/2.0/.
 name: SWI-Prolog
 scopeName: source.prolog
-fileTypes: [pl, pro]
+fileTypes: [pl, pro, prolog, swiplrc]
```

Comment:
While there are about 5k [`.prolog` files](https://github.com/search?q=extension%3Aprolog+NOT+dfhdjkfhfdj) on GitHub, I [couldn't find any `.swiplrc`](https://github.com/search?q=extension%3Aswiplrc+NOT+dfhdjkfhfdj). Where is this file extension used? Is there a typo in the extension maybe?

Reply:
`.swiprlrc` is the standard user startup (configuration) file name for Unix systems which contains SWI-Prolog code. For details, see [official SWI-Prolog documentation](http://www.swi-prolog.org/pldoc/man?section=initfile).

Reply:
Oh, I see. My search query was incorrect then. [That one](https://github.com/search?utf8=%E2%9C%93&q=filename%3A.swiplrc&type=) should be better. Thanks for the details!